### DISCUSSION TITLE
LaTeX Error: Missing \begin{document}.

### DISCUSSION THREAD

**Discussion by aravindb26** [Cyfrin/security-and-auditing-full-course-s23#17] [Q&A] [answered]
Can anyone help me with below error code please ?

```
Error producing PDF.
! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.1 {"
```

**Comment by cromewar** [comment]
@aravindb26 
Can you provide a little bit more context on this please? Or have you be able to solve this?

**Reply by aravindb26** [reply]
I got this error while I'm formatting my report to pdf format.

**Comment by PatrickAlphaC** [comment, marked as answer]
https://github.com/Cyfrin/security-and-auditing-full-course-s23/discussions/14#discussioncomment-7823216

**Reply by aravindb26** [reply]
pandoc report.md -o report.pdf --from markdown --template=eisvogel --listings
Error producing PDF.
! LaTeX Error: Missing \begin{document}.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.1 {"

I have different issue sir :)

**Comment by BikalpaRegmi** [comment]
hii aravindb256 did you solve this issue ?
If yes how

**Reply by EngrPips** [reply]
I see you resolve this, Great job sir
